pre-commit.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
```

Comment:
Is this file necessary as part of the pre-commit hooks in `.pre-commit-config.yaml`?